src/static/index.js

```diff
@@ -18,8 +18,9 @@
 
 import ReactDOM from 'react-dom';
 import { createBrowserHistory } from 'history';
+import Astarte from 'astarte-client';
```

Comment:
Remember this is just used as a namespace, the following code also imports the object
```
import { Client } from 'astarte-client';
```
but then it's not clear anymore which client we're refering to.
Do you really dislike having the class/object named `AstarteClient`?